deploy/agent/kindling-deploy.yml

```diff
@@ -26,6 +26,11 @@ spec:
             memory: 1Gi
           requests:
             memory: 300Mi
+        ports:
+        - containerPort: 9500
+          hostPort: 9500
+          name: http
```

Comment:
Should we decare hostPort when using `hostNetWork` policy

Reply:
No. I'll remove it.